api/v2/api.go

```diff
@@ -723,7 +723,7 @@ func (api *API) postSilencesHandler(params silence_ops.PostSilencesParams) middl
 	}
 
 	if sil.StartsAt.After(sil.EndsAt) || sil.StartsAt.Equal(sil.EndsAt) {
-		msg := "failed to create silence: start time must be equal or after end time"
+		msg := "failed to create silence: start time must neither be equal nor after end time"
```

Comment:
Sorry for the late review. Can you change the sentence to be affirmative? `start time must be greater than end time`

Reply:
Absolutely I'll do it now